From 816b7168c6380ec5a72416d3e0c52e22d891e2d9 Mon Sep 17 00:00:00 2001
From: Hiroyuki Ishii <ishii.hiroyuki002@jp.panasonic.com>
Date: Tue, 27 Dec 2022 14:44:14 +0900
Subject: [PATCH 01/17] Fix duplicated definition of bool

We'd like to use stdbool.h instead of defining bool type
duplicatedly to avoid build issue in future changes.

drm-backend backport from:
https://patchwork.yoctoproject.org/project/yocto/cover/20220425075954.10427-1-vasyl.vavrychuk@opensynergy.com/

Signed-off-by: Hiroyuki Ishii <ishii.hiroyuki002@jp.panasonic.com>
---
 psplash.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/psplash.h b/psplash.h
index 1c42ec7..9be44ba 100644
--- a/psplash.h
+++ b/psplash.h
@@ -34,10 +34,10 @@
 #include <sys/types.h>
 #include <termios.h>
 #include <unistd.h>
+#include <stdbool.h>
 
 typedef uint8_t  uint8;
 typedef uint16_t uint16;
-typedef int            bool;
 
 #ifndef FALSE
 #define FALSE 0
-- 
2.25.1

